ale_linters/markdown/languagetool.vim

```diff
@@ -0,0 +1,56 @@
+" Author: Vincent (wahrwolf [ät] wolfpit.net)
+" Description: languagetool for markdown files
+
+" Example output
+" 1.) Line 5, column 1, Rule ID:
+" SENT_START_CONJUNCTIVE_LINKING_ADVERB_COMMA[1]
+" Message: Did you forget a comma after a conjunctive/linking adverb?
+" Suggestion: Therefore,
+function! ale_linters#markdown#languagetool#Handle(buffer, lines) abort
+    let l:head_pattern = '^\v.+.\) Line (\d+), column (\d+), Rule ID. (.+)$'
+    let l:message_pattern = '^\vMessage. (.+)$'
+    let l:markers_pattern = '^\v *(\^+) *$'
+
+    let l:output = []
+
+    " Extract the header line first
+    let l:head_matches = []
+    call extend(l:head_matches, ale#util#GetMatches(a:lines, l:head_pattern))
+
+    " Extract all messages
+    let l:message_matches = []
+    call extend(l:message_matches, ale#util#GetMatches(a:lines, l:message_pattern))
```

Comment:
You could write `let l:message_matches = ale#util#GetMatches(a:lines, l:message_pattern)` and so on. Either way is fine.